examples/mnist/mnist_dist.py

```diff
@@ -0,0 +1,184 @@
+from argparse import ArgumentParser
+
+import torch
+from torch import nn
+import torch.nn.functional as F
+from torch.optim import SGD
+import torch.utils.data
+
+# === Added for distributed >>>
```

Comment:
Maybe we can remove these gards

Reply:
@vfdev-5 ok, I'll remove all comments such as "Added for distributed".